components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -4229,4 +4231,88 @@ private void checkSubscriptionAllowed(ApiTypeWrapper apiTypeWrapper)
                     apiTypeWrapper.getTier(), username));
         }
     }
+
+    /**
+     * This method is used to retrieve key manager configurations for tenant
+     * @param organization organization of the key manager
```

Comment:
The username param seems to be missing in the doc comment

Reply:
Fixed in cfc856942d5dcfb5dcdcc9eb904b5de6a6cdb495